lib/prism/debug.rb

```diff
@@ -114,8 +114,8 @@ def self.prism_locals(source)
                   AnonymousLocal
                 end
               end,
-              *params.keywords.reject(&:value).map(&:name),
-              *params.keywords.select(&:value).map(&:name)
+              *params.keywords.select { |kw| kw.is_a? OptionalKeywordParameterNode }.map(&:name),
+              *params.keywords.select { |kw| kw.is_a? RequiredKeywordParameterNode }.map(&:name),
```

Comment:
```suggestion
              *params.keywords.grep(OptionalKeywordParameterNode).map(&:name),
              *params.keywords.grep(RequiredKeywordParameterNode).map(&:name),
```

Since `Enumerable#grep` uses `===` to check for equality: https://ruby-doc.org/3.2.0/Enumerable.html#method-i-grep